README.md

```diff
@@ -55,18 +55,19 @@ Usage
 
 `cibuildwheel` runs inside a CI service. Supported platforms depend on which service you're using:
 
-|                 | Linux | macOS | Windows | Linux ARM | macOS ARM | Windows ARM |
-|-----------------|-------|-------|---------|-----------|-----------|-------------|
-| GitHub Actions  | ✅    | ✅    | ✅       | ✅        | ✅        | ✅²         |
-| Azure Pipelines | ✅    | ✅    | ✅       |           | ✅        | ✅²         |
-| Travis CI       | ✅    |       | ✅      | ✅        |           |             |
-| AppVeyor        | ✅    | ✅    | ✅      |           | ✅        | ✅²         |
-| CircleCI        | ✅    | ✅    |         | ✅        | ✅        |             |
-| Gitlab CI       | ✅    | ✅    | ✅      | ✅¹       | ✅        |             |
-| Cirrus CI       | ✅    | ✅    | ✅      | ✅        | ✅        |             |
+|                 | Linux | macOS | Windows | Linux ARM | macOS ARM | Windows ARM | iOS |
+|-----------------|-------|-------|---------|-----------|-----------|-------------|-----|
+| GitHub Actions  | ✅    | ✅    | ✅       | ✅        | ✅        | ✅²         | ✅³  |
+| Azure Pipelines | ✅    | ✅    | ✅       |           | ✅        | ✅²         |     |
+| Travis CI       | ✅    |       | ✅      | ✅        |           |             |     |
+| AppVeyor        | ✅    | ✅    | ✅      |           | ✅        | ✅²         |     |
+| CircleCI        | ✅    | ✅    |         | ✅        | ✅        |             |     |
+| Gitlab CI       | ✅    | ✅    | ✅      | ✅¹       | ✅        |             |     |
+| Cirrus CI       | ✅    | ✅    | ✅      | ✅        | ✅        |             |     |
```

Comment:
This only refers to GitHub Actions supporting building wheels for iOS.
There's probably little reason for other providers not to support this ?
It's probably already at least partially tested i this PR.

Reply:
Good point - I only added Github because that was the only one where I had run it personally; but it should work anywhere that there's a macOS runner that has iOS tooling, and we're validating most of those in CI. I'll update the table.